src/gsi/setupbend.f90

```diff
@@ -579,290 +580,295 @@ subroutine setupbend(obsLL,odiagLL, &
 
      if(ratio_errors(i) > tiny_r_kind)  then ! obs inside model grid
 
-        if (alt <= six) then
-           if (top_layer_SR >= 1) then ! SR exists for at least one layer. Check if obs is inside 
-              if (tpdpres(i)==ref_rad(top_layer_SR+1)) then !obs inside model SR layer
-                 qcfail(i)=.true.
-              elseif (tpdpres(i) < ref_rad(top_layer_SR+1)) then !obs below model close-to-SR layer
-                 qcfail(i)=.true.
-              elseif (tpdpres(i) >= ref_rad(top_layer_SR+1) .and. tpdpres(i) <= ref_rad(top_layer_SR+2)) then !source too close
-                 qcfail(i)=.true.
-              else !above
-              endif
-           endif
-
-!          check for SR in obs, will be updated in genstats. 
-           if ( data(igps,i) >= 0.03 .and. qc_layer_SR) then
-              kprof = data(iprof,i)
-              toss_gps_sub(kprof) = max (toss_gps_sub(kprof),data(igps,i))
-           endif
-        endif
-
-!       get pressure (in mb), temperature and moisture at obs location
-        call tintrp31(ges_lnprsi(:,:,1:nsig,:),dpressure,dlat,dlon,hob,&
+       if (alt <= six) then
+          if (top_layer_SR >= 1) then ! SR exists for at least one layer. Check if obs is inside 
+             if (tpdpres(i)==ref_rad(top_layer_SR+1)) then !obs inside model SR layer
+                qcfail(i)=.true.
+             elseif (tpdpres(i) < ref_rad(top_layer_SR+1)) then !obs below model close-to-SR layer
+                qcfail(i)=.true.
+             elseif (tpdpres(i) >= ref_rad(top_layer_SR+1) .and. tpdpres(i) <= ref_rad(top_layer_SR+2)) then !source too close
+                    qcfail(i)=.true.
+             else !above
```

Comment:
Will modifications be made for obs above model?  If not, should line 591 be removed?

Reply:
Based on replies from Kristen and Hui, the empty else block should be removed.  As such, the empty else block was removed from `src/gsi/setupbend.f90` at d199ed5.